fabric-tool-attribute-api-v1/src/main/java/net/fabricmc/fabric/impl/tool/attribute/handlers/VanillaToolsModdedBlocksToolHandler.java

```diff
@@ -51,12 +54,13 @@ public ActionResult isEffectiveOn(Tag<Item> tag, BlockState state, ItemStack sta
 		return ActionResult.PASS;
 	}
 
+	@NotNull
 	@Override
 	public TypedActionResult<Float> getMiningSpeedMultiplier(Tag<Item> tag, BlockState state, ItemStack stack, LivingEntity user) {
 		if (!(stack.getItem() instanceof DynamicAttributeTool)) {
 			ToolManagerImpl.Entry entry = ToolManagerImpl.entryNullable(state.getBlock());
 
-			if (entry != null && entry.getMiningLevel(tag) >= 0 && tag.contains(stack.getItem())) {
+			if (entry != null && entry.getMiningLevel(tag) >= 0) {
```

Comment:
Is there a reason the tag check is dropped here?

Reply:
Mentioned in the first message